acme/client.go

```diff
@@ -354,6 +385,33 @@ func (c *Client) RenewCertificate(cert CertificateResource, bundle bool) (Certif
 }
 
 // Looks through the challenge combinations to find a solvable match.
+// Then presents the challenges in series and returns.
+func (c *Client) presentChallenges(challenges []authorizationResource) (map[string]Challenge, map[string]error) {
```

Comment:
@mholt, is there a way I can just return a single map here? Challenge has a `RemoteError` but no `error`.